src/App.tsx

```diff
@@ -1,11 +1,11 @@
 // React Imports
```

Comment:
you can set up this thing called husky that will let you add event hooks (not react hooks) to your repo, so you can run a  script before committing, https://dev.to/botreetechnologies/setting-up-husky-pre-commit-hook-with-eslint-prettier-and-lint-staged-for-react-and-react-native-d05